src/components/app.js

```diff
@@ -41,6 +42,7 @@ let App = class App extends React.PureComponent {
 
     this.map.on('load', () => {
       this.map.addControl(this.draw);
+      this.map.resize();
```

Comment:
What's this for?

Reply:
See the OP: 

> The way I put together the entry file seems to have messed up the timing between when the App component is mounted and when Mapbox GL JS decides how big its container is, so I added a map.resize() inside componentDidMount. Please shout at me if there's a more effective way to do this!

Happy to switch to another approach if there's one you prefer!

Reply:
Oh I missed that! This works for me. Because its purpose is a little unclear, do you mind adding a little inline comment explaining its existence?